src/test/react/SnapshotComponentTesting/__snapshots__/PostListingDisplay.test.js.snap

```diff
@@ -274,9 +274,9 @@ exports[`PostListingDisplay renders correctly 1`] = `
         Lease End Date: 
       </label>
       <input
-        defaultValue="2020-08-05"
+        defaultValue="2020-08-07"
         id="endDate"
-        min="2020-08-05"
+        min="2020-08-07"
```

Comment:
It is an auto generated snapshot that gets made when running the tests. I've read that these snapshot tests should be committed into github. It gets changed because the date input field has it's min set as today's date, so when I update the tests, they will change to today's date. I'm don't think that this is what I want, but I'm also not sure how to fix it